src/components/QRCode.tsx

```diff
@@ -54,10 +72,14 @@ export default function ({ derivativeAddress, tokenId }: QRCodeProps) {
   return (
     <a
       href={url}
-      ref={ref}
       className={classNamesToString(qrCodeContainer, 'custom-qr-code')}
       target="_blank"
       rel="noopener noreferrer"
-    />
+    >
+      <div ref={ref} />
```

Comment:
What does this `div` do?

Reply:
`QRCodeStyling` generates canvas with QR into that div block. It is needed because generating QR will overwrite all children in the element. So not to lose the `logo`, we need to put it near it in a separate block